config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleDocumentBuilder.java

```diff
@@ -61,4 +81,18 @@ private Element buildEntityItem(final Entity entity, final Document document) {
                 createElementWithChildren(document, RESOURCE, entity.getXml())
         );
     }
+
+    /**
+     * Remove environment entities and folders from the entity list. Environment entities and Folders should not be
+     * part of DELETE bundle.
+     *
+     * @param entities list of entities to be included in the delete bundle
+     */
+    private void removeEntitiesForDeleteBundle(final List<Entity> entities) {
```

Comment:
you can do this in the BundleEntityBuilder itself in createDeleteBundle method

Reply:
Fixed.